sdk/test/Services/DynamoDBv2/UnitTests/Custom/DynamoDBEntryConversionTests.cs

```diff
@@ -22,6 +24,229 @@ public void ValidateAllConvertersAreRegisteredForConversionV2()
 
+        [DataTestMethod]
+        [DynamicData(nameof(V1PrimitiveData))]
+        public void V1_ConvertToEntry_Primitives(object value, DynamoDBEntry expectedPrimitive)
+        {
+            var actualPrimitive = DynamoDBEntryConversion.V1.ConvertToEntry(value.GetType(), value);
+            Assert.AreEqual(expectedPrimitive, actualPrimitive);
+        }
+
+        [DataTestMethod]
+        [DynamicData(nameof(V1PrimitiveData))]
+        public void V1_ConvertFromEntry_Primitives(object expectedValue, DynamoDBEntry primitive)
+        {
+            var actualValue = DynamoDBEntryConversion.V1.ConvertFromEntry(expectedValue.GetType(), primitive);
+            Assert.AreEqual(expectedValue, actualValue);
+        }
+
+        [DataTestMethod]
+        [DynamicData(nameof(V2PrimitiveData))]
+        public void V2_ConvertToEntry_Primitives(object value, DynamoDBEntry expectedPrimitive)
+        {
+            var actualPrimitive = DynamoDBEntryConversion.V2.ConvertToEntry(value.GetType(), value);
+            Assert.AreEqual(expectedPrimitive, actualPrimitive);
+        }
+
+        [DataTestMethod]
+        [DynamicData(nameof(V2PrimitiveData))]
+        public void V2_ConvertFromEntry_Primitives(object expectedValue, DynamoDBEntry primitive)
+        {
+            var actualValue = DynamoDBEntryConversion.V2.ConvertFromEntry(expectedValue.GetType(), primitive);
+            Assert.AreEqual(expectedValue, actualValue);
+        }
+
+        [DataTestMethod]
+        [DynamicData((nameof(DynamoDBEntryConversions)))]
+        public void ConvertToEntry_MemoryStream(DynamoDBEntryConversion conversion)
+        {
+            var memoryStream = new MemoryStream(ByteArray);
+            var entry = conversion.ConvertToEntry(memoryStream);
+            Assert.AreEqual(new Primitive(ByteArray), entry);
+        }
+
+        [DataTestMethod]
+        [DynamicData((nameof(DynamoDBEntryConversions)))]
+        public void ConvertFromEntry_MemoryStream(DynamoDBEntryConversion conversion)
+        {
+            var entry = new Primitive(ByteArray);
+            var memoryStream = conversion.ConvertFromEntry<MemoryStream>(entry);
+            CollectionAssert.AreEqual(ByteArray, memoryStream.ToArray());
+        }
+
+        [DataTestMethod]
+        [DynamicData((nameof(DynamoDBEntryConversions)))]
+        public void ConvertToEntry_DateTime(DynamoDBEntryConversion conversion)
+        {
+            var dateTime = new DateTime(2024, 07, 03, 01, 31, 47, DateTimeKind.Utc);
+            var entry = conversion.ConvertToEntry(dateTime);
+            Assert.AreEqual(new Primitive("2024-07-03T01:31:47.000Z", false), entry);
+        }
+
+        [DataTestMethod]
+        [DynamicData((nameof(DynamoDBEntryConversions)))]
+        public void ConvertFromEntry_DateTime(DynamoDBEntryConversion conversion)
+        {
+            // DynamoDBEntryConversion returns local DateTime by default:
+            // https://github.com/aws/aws-sdk-net/issues/1450
```

Comment:
This is no longer true with the addition of this merged PR which will have binaries in Preview 5: https://github.com/aws/aws-sdk-net/pull/3572

Because of this,  `actualDateTime` could be replaced with utcDateTime alone. Although the code below will work with the ToUniversalTime, it won't actually do anything.

Reply:
Nice, updated.